libs/ui/src/lib/button/button.stories.tsx

```diff
@@ -38,13 +38,27 @@ export const Intent = IntentExamples.bind({});
 
 const SizeExamples = () => (
   <div className="grid gap-4">
-    <Button href="https://weshipit.today" size="sm">
+    <Button as={'a'} href="https://weshipit.today" size="sm">
       filled
     </Button>
-    <Button href="https://weshipit.today" size="md">
+    <Button as={'a'} href="https://weshipit.today" size="md">
       filled
     </Button>
   </div>
 );
-
 export const Size = SizeExamples.bind({});
+
+const TypeExamples = () => (
+  <div className="grid gap-4">
+    <Button as={'a'} href="https://weshipit.today" size="sm">
```

Comment:
it doesnt work 
![image](https://github.com/flexbox/weshipit.today/assets/360936/a963ab4f-50ad-460c-afda-33f2684280fc)